src/ssh/cockpitsshrelay.c

```diff
@@ -458,7 +459,15 @@ set_knownhosts_file (CockpitSshData *data,
         }
     }
 
-  /* TODO: Check more sources of known hosts here if !host_known */
+  /* check ~/.ssh/known_hosts, unless we are running as a system user ($HOME == "/"); this is
```

Comment:
Ugh, that's embarassing. I thought this was the cockpitauthoptions file. Please disregard.